lib/carbonite/outbox.ex

```diff
@@ -14,6 +14,7 @@ defmodule Carbonite.Outbox do
   import Ecto.Changeset
 
   @primary_key false
+  @schema_prefix "carbonite_default"
```

Comment:
do you think it makes sense to share the `"carbonite_default"` strings in a macro somewhere?